schemas/fluidity_options.rnc

```diff
@@ -3562,12 +3662,106 @@ prognostic_detector_options = detector_options_enabled_default
 diagnostic_detector_options = detector_options_enabled_default
 prescribed_detector_options = detector_options_disabled_default
 
+# Specify whether or not values for this field are interpolated to particle locations.
+particle_options_enabled_default = 
+   (
+      ## Specify what is added to particles
+      element particles {
+         (
+            ## This field is included at each particle location.
+            element include_in_particles {
+	       element store_old_field {
+	         ## Store values of field at preivous timestep/non-linear iteration, for use when updating particle attributes.
```

Comment:
Looks like this docstring is in the wrong place - it needs to be above the element to appear in diamond.

Reply:
Also typo... previous